tests/com/rcv/TabulatorTests.java

```diff
@@ -75,9 +75,10 @@ static boolean fileCompare(String path1, String path2) {
         // both files have content so compare it
         if (!line1.equals(line2)) {
           // report inequality and keep processing in case there are more inequalities
-          Logger.log(Level.SEVERE, "files are not equal:\n%s\n%s", line1, line2);
+          Logger.log(Level.SEVERE, "files are not equal (%d)\n%s\n%s", currentLine, line1, line2);
           result = false;
```

Comment:
Add a break here to avoid log spam if lots of lines aren't equal?

Reply:
Added a counter which caps output at 10 lines